extensions/tajo/tajo_env.sh

```diff
@@ -0,0 +1,135 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains environment-variable overrides to be used in conjunction
+# with bdutil_env.sh in order to deploy a Hadoop cluster with HBase installed
+# and configured.
+# Usage: ./bdutil -e tajo deploy
+
+# URIs of Tajo tarball to install.
+# Recommended Tajo version: Apache Tajo 0.11.0 or higher (eg. gs://your_bucket/tajo-x.xx.x.tar.gz)
+TAJO_TARBALL_URI='gs://tajo_release/tajo.tar.gz'
```

Comment:
Who owns the tajo_release bucket and who can write to it? Would it make sense for us to hose the distribution in a bucket of our own? The downside is of course the cost of a PR for a new version, but the upside is that our project is billed for any data transfer.

It looks like we host a hama distribution tarball, but the flink tarball is controlled by the flink project so it's up to you.

Either way, I'd prefer some sort of identifier for the version (even if there's always one called tajo-current.tar.gz also having tajo-0.10.x.tar.gz allows people to pin their deploys to a specific version).